### ISSUE TITLE
Theme Resets upon restart

### ISSUE THREAD

**Issue by Crocodile73** [namazso/SecureUxTheme#162]
Had this issue for a while. Apply a theme and then once I restart or shutdown, the windows default theme is applied. Tried all the normal steps, reinstalling, check the dll is in system32 etc. The only thing I found was that if I apply a theme, then sign out and back in, the theme will stick on restart or shutdown. Windows 22H2 x64. Using latest secureuxtheme 2.2.0

**Comment by sajko-srgb** [comment]
i have same issue

**Comment by Crocodile73** [comment]
> i have same issue

Does it stick, if you log out and back in, and then restart?

**Comment by djsolarix** [comment]
> > i have same issue
> 
> Does it stick, if you log out and back in, and then restart?

That seems to be the only way to get it to work. 

Apply theme, restart, log in, fails to load. Log out, Log In, re apply theme then it works, but will fail again once you restart.

**Comment by rodolfol95** [comment]
Yes been having the same issue themes are broken after a restart almost always had to reapply

**Comment by djsolarix** [comment]
Problem still persists on 23H2.
